test/smiley_examples.jl

```diff
@@ -0,0 +1,220 @@
+# examples from
+# M. W. Smiley and C. Chun, J. Comput. Appl. Math. **137**, 293 (2001).
+# https://doi.org/10.1016/S0377-0427(00)00711-1
+
+module SmileyExample22
+
+using IntervalArithmetic
+using StaticArrays
+
+const title = "Smiley and Chun (2001), Example 2.2"
+
+f(x) = SVector(
+    x[1]^2 + 4 * x[2]^2 - 4,
+    x[2] * (x[1] - 1.995) * (x[2] - x[1]^2) * (x[2] - x[1] + 1)
+)
+
+# contains all 8 reported roots
+const region = IntervalBox(-3..3, -3..3)
+
+# only three roots are reported explicitely:
+# (2, 0) and (1.995, ±0.071)
+
+end
+
+module SmileyExample52
+
+using IntervalArithmetic
+using StaticArrays
+
+const title = "Smiley and Chun (2001), Example 5.2"
+
+const c0  =  1
+const a0  =  0.5
+const b0  =  0.5
+const d01 =  0.2
+const d02 = -0.7
+const r1  =  1
+const r2  =  2
+const m   =  3
+const θs  = [i * pi / m for i in 1:m]
+_v(θ) = -(a0 * cos(θ) + b0 * sin(θ)) / c0
+const vs = _v.(θs)
+
+_a(θ, v) = b0 * v - c0 * sin(θ)
+const as = _a.(θs, vs)
+_b(θ, v) = c0 * cos(θ) - a0 * v
+const bs = _b.(θs, vs)
+_c(θ) = a0 * sin(θ) - b0 * cos(θ)
+const cs = _c.(θs)
+_d(c) = d01 * c / c0
+const ds = _d.(cs)
+
+f(x) = SVector(
+    (x[1]^2 + x[2]^2 + x[3]^2 - r1^2) * (x[1]^2 + x[2]^2 + x[3]^2 - r2^2),
+    (a0 * x[1] + b0 * x[2] + c0 * x[3] - d01) *
+        (a0 * x[1] + b0 * x[2] + c0 * x[3] - d02),
+    prod(as * x[1] .+ bs * x[2] .+ cs * x[3] .- ds)
+)
+
+# contains all 24 reported roots
+const region = IntervalBox(-3..3, -3..3, -3..3)
+
+const known_roots = [
+    IntervalBox(-1.933009 ± 1e-6, -0.300000 ± 1e-6,  0.416504 ± 1e-6),
+    IntervalBox(-1.701684 ± 1e-6,  0.000000 ± 1e-6,  1.050842 ± 1e-6),
+    IntervalBox(-1.258803 ± 1e-6,  1.360696 ± 1e-6, -0.750946 ± 1e-6),
+    IntervalBox(-1.044691 ± 1e-6, -1.589843 ± 1e-6,  0.617267 ± 1e-6),
+    IntervalBox(-0.996600 ± 1e-6,  1.726162 ± 1e-6, -0.164780 ± 1e-6),
+
+    IntervalBox(-0.951026 ± 1e-6, -0.300000 ± 1e-6, -0.074486 ± 1e-6),
+    IntervalBox(-0.800000 ± 1e-6, -0.000000 ± 1e-6,  0.600000 ± 1e-6),
+    IntervalBox(-0.776373 ± 1e-6, -1.344718 ± 1e-6,  1.260546 ± 1e-6),
+    IntervalBox(-0.717665 ± 1e-6,  0.423418 ± 1e-6, -0.552876 ± 1e-6),
+    IntervalBox(-0.592072 ± 1e-6, -0.805884 ± 1e-6, -0.001021 ± 1e-6),
+
+    IntervalBox(-0.499927 ± 1e-6,  0.865900 ± 1e-6,  0.017013 ± 1e-6),
+    IntervalBox(-0.360640 ± 1e-6, -0.624646 ± 1e-6,  0.692643 ± 1e-6),
+    IntervalBox( 0.082249 ± 1e-6, -0.962075 ± 1e-6, -0.260086 ± 1e-6),
+    IntervalBox( 0.085220 ± 1e-6,  0.367221 ± 1e-6, -0.926221 ± 1e-6),
+    IntervalBox( 0.453788 ± 1e-6,  0.785984 ± 1e-6, -0.419886 ± 1e-6),
+
+    IntervalBox( 0.464511 ± 1e-6, -0.804557 ± 1e-6,  0.370022 ± 1e-6),
+    IntervalBox( 0.511026 ± 1e-6, -0.300000 ± 1e-6, -0.805513 ± 1e-6),
+    # the following two roots are suspect, first column probably reported in error
```

Comment:
Do you mean that the error is in the original paper?

Reply:
yes, i think so. as you can see the two commented roots contain values from the three following roots. i think they made an error in the first column by first skipping 0.537839 and then repeating 0.869521.